cookie_crimes.py

```diff
@@ -26,9 +26,9 @@
     CHROME_CMD = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
     USER_DATA_DIR = "$HOME/Library/Application Support/Google/Chrome"
 
-elif sys.platform.startswith == "win":
-    CHROME_CMD = "chrome.exe"
-    USER_DATA_DIR = r"%LOCALAPPDATA%\Google\Chrome\User Data"
+elif sys.platform.startswith("win"):
+    CHROME_CMD = "\"C:\\Program Files (x86)\\Google\\Chrome\Application\\chrome.exe\""
```

Comment:
I think you should use this kind of discovery method to ensure the path is good: 
https://stackoverflow.com/a/44588190

Reply:
Completely agree, hardcoded path is bad. However I'm short in time and just wanted to get it working on my pretty standard Windows 10 installation. There's also the potential problem of having to escalate privileges to fetch this registry. But don't quote me on that one.

This patch also fixes other problems when running on Windows so I was hopping for it to land and be further improved upon by someone else with a bit more time.